modules/distr-tracing-otel-config-collector.adoc

```diff
@@ -0,0 +1,389 @@
+<5> The TLS server side configuration. See the OTLP receiver configuration section for more details.
+
+==== Zipkin Receiver
+
+The Zipkin receiver ingests data in the Zipkin v1 and v2 formats.
+
+* Support level: link:https://access.redhat.com/support/offerings/techpreview[Technology Preview]
+* Supported signals: traces
+
+.OpenTelemetry Collector custom resource with enabled Zipkin receiver
+[source,yaml]
+----
+  config: |
+    receivers:
+      zipkin:
+        endpoint: 0.0.0.0:9411 <1>
+        tls: <2>
+
+    service:
+      pipelines:
+        traces:
+          receivers: [zipkin]
+----
+<1> The Zipkin HTTP endpoint. If omitted, the default `+0.0.0.0:9411+` is used.
+<2> The TLS server side configuration. See the OTLP receiver configuration section for more details.
+
+
+=== Processors
+
+==== Resource Detection processor
+
+The Resource Detection processor is designed to identify host resource details in alignment with OpenTelemetry's resource semantic standards. Using this detected information, it can add or replace the resource values in telemetry data.
+
+* Support level: link:https://access.redhat.com/support/offerings/techpreview[Technology Preview]
+* Supported signals: traces, metrics
+
+.{product-title} permissions required for the Resource Detection processor
+[source,yaml]
+----
+kind: ClusterRole
+metadata:
+  name: otel-collector
+rules:
+- apiGroups: ["config.openshift.io"]
+  resources: ["infrastructures", "infrastructures/status"]
+  verbs: ["get", "watch", "list"]
+----
+
+.OpenTelemetry Collector using the Resource Detection processor
+[source,yaml]
+----
+  config: |
+    processor:
+      resourcedetection:
+        detectors: [openshift]
+        override: true
+    service:
+      pipelines:
+        traces:
+          processors: [resourcedetection]
+        metrics:
+          processors: [resourcedetection]
+----
+
+=== Exporters
+
+==== OTLP exporter
+
+The OTLP gRPC exporter exports data using the OpenTelemetry protocol (OTLP).
+
+* Support level: link:https://access.redhat.com/support/offerings/techpreview[Technology Preview]
+* Supported signals: traces, metrics
+
+.OpenTelemetry Collector custom resource with an enabled OTLP exporter
+[source,yaml]
+----
+  config: |
+    exporters:
+      otlp:
+        endpoint: tempo-ingester:4317 <1>
+        tls: <2>
+          ca_file: ca.pem
+          cert_file: cert.pem
+          key_file: key.pem
+        headers: <3>
+          X-Scope-OrgID: "dev"
+    service:
+      pipelines:
+        traces:
+          exporters: [otlp]
+        metrics:
+          exporters: [otlp]
+----
+<1> The OTLP gRPC endpoint. If the `+https://+` scheme is used, then client transport security is enabled and overrides the `insecure` setting in the `tls`.
+<2> The client side TLS configuration. Defines paths to TLS certificates.
+<3> Headers are sent for every RPC performed during an established connection.
+
+==== OTLP HTTP exporter
```

Comment:
[Anchoring in module files](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-in-module-files)
Missing anchor IDs

Reply:
Thank you. Fixed.